docker-compose.env.sample

```diff
@@ -38,3 +38,4 @@ BIG_QUERY_DATASET=
 CLOUD_STORAGE_API_JSON_KEY=
 CLOUD_STORAGE_BUCKET=
 GOOGLE_CLOUD_PLATFORM_PROJECT_ID=
+FEATURE_UPLOAD_DOCUMENTS=true
```

Comment:
While you're adding to this file, could you put these in alphabetical order, please?

Reply:
Will do this as part of the upload docs work I do, thanks for the one liner command for this 😄